services/sourceforge/sourceforge-status.service.js

```diff
@@ -0,0 +1,39 @@
+import Joi from 'joi'
+import BaseSourceForgeService from './sourceforge-base.js'
+
+const schema = Joi.object({
+  status: Joi.string().required(),
```

Comment:
What does this status value mean? What values could it have other than "active"?

Reply:
I don't know because SourceForge doesn't publish full documentation of their api. This status system is not documented anywhere.  More, I did not find any project with a status other than ''active''

Reply:
I think if we're going to have a badge for it, we need to know what it is.

For example, how do we know "monitoring" is the right category for this if we don't really know what the value means?